templates/16-04/azure.html

```diff
@@ -102,31 +105,13 @@ <h4>
         If you are looking to upgrade your existing Ubuntu 16.04 LTS Azure instances you can <a class="p-link--external" href="https://azuremarketplace.microsoft.com/uk-ua/marketplace/apps/Canonical.UbuntuAdvantageSupport?tab=Overview">enable the Ubuntu Advantage subscription directly from the Azure marketplace</a>.
       </p>
       <p>
-        Please note that this will enable Ubuntu Advantage for all Ubuntu instances under the selected subscription.
+        <a class="p-link--external" href="https://ubuntu.com/tutorials/purchasing-and-applying-16-04-esm-from-the-azure-marketplace">See a tutorial for this process</a>. Please note that this will enable Ubuntu Advantage for all Ubuntu instances under the selected subscription.
```

Comment:
This link isn't external 

```suggestion
        <a href="/tutorials/purchasing-and-applying-16-04-esm-from-the-azure-marketplace">See a tutorial for this process</a>. Please note that this will enable Ubuntu Advantage for all Ubuntu instances under the selected subscription.
```